packages/core/src/ray-exports.scss

```diff
@@ -1,5 +1,10 @@
 @import 'global/variables';
 
+$aspect-ratios: ();
```

Comment:
why is this part needed?

Reply:
Took from doc examples. I believe you need to initialize it to something so can `append` to it